libs/blocks/aside/aside.js

```diff
@@ -84,6 +91,25 @@ function decorateVideo(container) {
   container.classList.add('has-video');
 }
 
+function formatPromoButton(el) {
+  if (!(el.classList.contains('promobar') && el.classList.contains('popup'))) return;
+  const actionArea = el.querySelectorAll('.action-area');
+  [...actionArea].forEach((aa) => {
+    const buttons = aa.querySelectorAll('.con-button');
+    [...buttons].forEach((btn) => {
+      if (!btn.classList.contains('outline')) btn.classList.add('fill');
+    });
+  });
+}
+
+function addCloseButton(el) {
+  const closeBtn = createTag('button', { class: 'promo-close close-sticky-section', 'aria-label': 'Close' }, closeSvg);
+  el.querySelector('.foreground').appendChild(closeBtn);
+  closeBtn.addEventListener('click', (e) => {
+    e.target.closest('.section').classList.add('close-sticky-section');
```

Comment:
I see the same class `close-sticky-section` being added to both close button and parent wrapper when the promo is closed. Using the same class might produce side-effects if we ever plan to use it for CSS purposes or as JS selector. Additionally, what's the usage for this class? I've only seen it in tests.

Reply:
@narcis-radu  I have made the suggested change and remove the class from the close button (It was not being monitored hence it will not be required at the button).

Reply:
Why is `close-sticky-section` added even when it is not a sticky-section?

Reply:
@chrischrischris Popup promo will only be used with sticky section.